lib/chat.js

```diff
@@ -28,25 +17,6 @@ var Chat = {
 
 	send_team: function(player, to_team, text){
 		Logger.debug('Chat.send_team');
-		if (!player.initials){
```

Comment:
Isn't this the kind of stuff you should be warning in the log? Or are you relying on the exceptions?